src/azure-cli/azure/cli/command_modules/storage/commands.py

```diff
@@ -501,15 +501,18 @@ def get_custom_sdk(custom_module, client_factory, resource_type=ResourceType.DAT
                                                                cf_mgmt_file_shares,
                                                                resource_type=ResourceType.MGMT_STORAGE),
                             resource_type=ResourceType.MGMT_STORAGE, min_api='2019-04-01') as g:
+        from ._transformers import transform_share_rm_output
         g.custom_command('create', 'create_share_rm')
         g.command('delete', 'delete', confirmation=True)
         g.custom_command('exists', '_file_share_exists', transform=create_boolean_result_output_transformer('exists'))
         g.custom_command('list', 'list_share_rm')
-        g.show_command('show', 'get')
+        g.show_command('show', 'get', transform=transform_share_rm_output)
         g.generic_update_command('update', setter_name='update', setter_arg_name='file_share',
                                  custom_func_name='update_share_rm')
         g.custom_command('stats', 'get_stats', transform=lambda x: getattr(x, 'share_usage_bytes'))
         g.custom_command('restore', 'restore_share_rm')
+        g.custom_command('snapshot', 'snapshot_share_rm', min_api='2020-08-01-preview',
```

Comment:
could you add preview tag for new commands?

Reply:
Added for new command and new parameter.